superset-frontend/src/components/TableView/TableView.tsx

```diff
@@ -49,6 +49,7 @@ export interface TableViewProps {
   isPaginationSticky?: boolean;
   showRowCount?: boolean;
   scrollTable?: boolean;
+  scrollTopOnPagination?: boolean;
```

Comment:
I wonder if we should even expose a property to configure this behavior. My fear is that we would allow different behaviors throughout the app. We could follow the same approach we did with the Select component and not expose this property until a valid use case appears. WDYT?